assets/sass/modules/_email-factor.scss

```diff
@@ -0,0 +1,12 @@
+@mixin email-factor {
+  .resend-email-infobox {
+    margin-bottom: 15px;
```

Comment:
Are we already using standard Odyssey spacing here or not yet?

Reply:
no we don't